package.json

```
@@ -2,8 +2,10 @@
   "version": "1.0.0",
   "scripts": {
     "start": "vite",
+    "start:ipfs": "HASH_ROUTING=true vite --base ./",
```

```suggestion
    "start:ipfs": "HASH_ROUTING=1 vite --base ./",
```
If we're not reading the value, this is usually the convention

---

79463006